### ISSUE TITLE
Support Request:  SwitchBot API Has No Devices With Cloud Services Enabled

### ISSUE THREAD

**Issue by deflao** [OpenWonderLabs/homebridge-switchbot#1121]
### Describe Your Problem

I can't control any of my switchbot devices from Home app, but it functions fine in Switchbot app.

### Relevant log output

```shell
[22/11/2024, 22:06:31] [SwitchBot] [DEBUG] Platform Config pushRate Not Set: undefined
[22/11/2024, 22:06:31] [SwitchBot] [DEBUG] Platform Config maxRetries Not Set: undefined
[22/11/2024, 22:06:31] [SwitchBot] [DEBUG] Platform Config delayBetweenRetries Not Set: undefined
[22/11/2024, 22:06:31] [SwitchBot] [DEBUG] Got devices: {"statusCode":190,"body":{},"message":"Requests reached the daily limit"}
[22/11/2024, 22:06:31] [SwitchBot] [DEBUG] statusCode: 200
[22/11/2024, 22:06:31] [SwitchBot] [DEBUG] response: {"statusCode":190,"body":{},"message":"Requests reached the daily limit"}
[22/11/2024, 22:06:31] [SwitchBot] [DEBUG] SwitchBot Device Config Not Set: undefined
[22/11/2024, 22:06:31] [SwitchBot] [DEBUG] SwitchBot API Has No Devices With Cloud Services Enabled
[22/11/2024, 22:06:31] [SwitchBot] [DEBUG] IR Device Config Not Set: undefined
[22/11/2024, 22:23:26] [SwitchBot] Initializing SwitchBot platform...
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Using Platform Config Logging: debug
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Platform Config refreshRate Not Set: undefined
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Platform Config: {"platform":"Resideo","logging":"debug"}
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Plugin Version: 4.1.4
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Finished initializing platform: SwitchBot
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Verifying Config
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Config OK
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Platform BLE is not enabled
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Loading accessory from cache: Curtain
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Loading accessory from cache: Sliding Door
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Loading accessory from cache: Curtain - Left
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Loading accessory from cache: Curtain - Right
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Loading accessory from cache: Fan
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Loading accessory from cache: Humidifier
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Loading accessory from cache: Humidifier E2
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Loading accessory from cache: Curtain 2A
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Executed didFinishLaunching callback
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Retry Count: 0
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Max Retries: undefined
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Delay Between Retries: undefined
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Platform Config updateRate Not Set: undefined
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Platform Config pushRate Not Set: undefined
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Platform Config maxRetries Not Set: undefined
[22/11/2024, 22:23:28] [SwitchBot] [DEBUG] Platform Config delayBetweenRetries Not Set: undefined
[22/11/2024, 22:23:30] [SwitchBot] [DEBUG] Got devices: {"statusCode":190,"body":{},"message":"Requests reached the daily limit"}
[22/11/2024, 22:23:30] [SwitchBot] [DEBUG] statusCode: 200
[22/11/2024, 22:23:30] [SwitchBot] [DEBUG] response: {"statusCode":190,"body":{},"message":"Requests reached the daily limit"}
[22/11/2024, 22:23:30] [SwitchBot] [DEBUG] SwitchBot Device Config Not Set: undefined
[22/11/2024, 22:23:30] [SwitchBot] [DEBUG] SwitchBot API Has No Devices With Cloud Services Enabled
[22/11/2024, 22:23:30] [SwitchBot] [DEBUG] IR Device Config Not Set: undefined
```

### Config for homebridge-switchbot

```shell
{
    "name": "SwitchBot",
    "credentials": {
        "token": "***",
        "secret": "***",
        "notice": "Keep your Token & Secret a secret!"
    },
    "options": {
        "logging": "debug"
    },
    "platform": "SwitchBot"
}
```

### Device & Model

Raspberry Pi 4

### Node.js Version

v22.11.0

### NPM Version

N/A

### Homebridge Version

v1.8.5

### Homebridge SwitchBot Plugin Version

v4.1.4

### Homebridge Config UI X Plugin Version

v4.63.0

### Operating System

Raspbian GNU/Linux Bullseye (11)

**Comment by semih-akbas** [comment]
+1

**Comment by bart015** [comment]
Same issue

**Comment by deflao** [comment]
Fixed with v4.1.5
